PhoneBook-App/UserInput.cs

```diff
@@ -0,0 +1,34 @@
+﻿
```

Comment:
I noticed that a few of your classes don't have namespaces. While this is neglectible in small projects, you definitely don't want to create this habit. Namespaces are essential for the maintainability of code as projects grow larger.